docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -109,6 +112,28 @@ When a Global Admin invites a user to join an account, the Global Admin selects
   - Has Namespace Admin permissions for each Namespace created by that user
 - **Read-Only:** Can only read information
 
+:::note Default Role
+
+By default, the first user assigned to your account is granted the **Account Owner** role. If your account does not have an Account Owner, please [contact Support](https://temporalsupport.zendesk.com/) to assign the appropriate individual to this role.
```

Comment:
When the account is created, the initial user who logs in is automatically assigned the Account Owner role. If your account does not have an Account Owner, please reach out to support to have the appropriate individual assigned to this role.

Reply:
Reworded.